src/routes/auth/connexion/+page.svelte

```diff
@@ -24,92 +26,141 @@
+                {@html informationLineIcon}
+              </div>
+              <div>DORA passe à Inclusion Connect&nbsp!</div>
+            </h3>
+            <p class="legend mb-s16 text-gray-text">
+              <strong>Si vous aviez un ancien compte DORA,</strong> vous pouvez simplement
+              vous inscrire à Inclusion Connect avec la même adresse e-mail pour
+              retrouver vos données.
+            </p>
+            <p class="legend mb-s16 text-gray-text">
+              <strong>Si vous avez déjà un compte Inclusion Connect,</strong> vous
+              pouvez l‘utiliser pour accéder plus facilement à DORA.
+            </p>
           </div>
-          <div>DORA passe à Inclusion Connect&nbsp!</div>
-        </h4>
-        <div class="legend mb-s16 text-gray-text">
-          Si vous avez un Compte DORA, il vous suffit de créer un compte
-          Inclusion Connect avec la même adresse e-mail afin de retrouver les
-          mêmes droits et données.
-        </div>
-      </div>
 
-      <p class="mt-s24 mb-s24" />
-      <div class="mb-s40 text-center">
-        <a
-          class="inline-block"
-          href="/auth/ic-connect?next={encodeURIComponent(nextPage)}{loginHint}"
-        >
-          <div
-            class="mx-auto flex items-center justify-center rounded-full bg-france-blue p-s6 pr-s24 text-f16 text-white transition-colors hover:bg-[#2323FF]"
-          >
-            <span class="h-[62px] w-[62px] rounded-full bg-white p-s6">
-              <img src={logoIC} alt="" class="max-w-none" />
-            </span>
-            <div class="ml-s16 text-left">
-              Se connecter avec <strong class="whitespace-nowrap">
-                Inclusion Connect
-              </strong>
+          <div class="text-center">
+            <a
+              class="mb-s24 inline-block"
+              href="/auth/ic-connect?next={encodeURIComponent(
+                nextPage
+              )}{loginHint}"
+            >
+              <div
+                class="mx-auto mt-s24 flex items-center justify-center rounded-full bg-france-blue p-s6 pr-s24 text-f16 text-white transition-colors hover:bg-[#2323FF]"
+              >
+                <span class="h-[62px] w-[62px] rounded-full bg-white p-s6">
+                  <img src={logoIC} alt="" class="max-w-none" />
+                </span>
+                <div class="ml-s16 text-left">
+                  S‘identifier avec <strong class="whitespace-nowrap">
+                    Inclusion Connect
+                  </strong>
+                </div>
+              </div>
+            </a>
+
+            <div class="text-center">
+              <a
+                class="text-magenta-cta underline"
+                target="_blank"
+                title="Ouverture dans une nouvelle fenêtre"
+                rel="noopener"
+                href="https://aide.dora.fabrique.social.gouv.fr/fr/category/inscription-et-gestion-du-compte-ha8m5b/"
```

Comment:
```suggestion
                href="https://aide.dora.inclusion.beta.gouv.fr/fr/category/inscription-et-gestion-du-compte-ha8m5b/"
```
ça va sans doute se faire tout seul lors du merge de main, mais à vérifier